devel/libSDL2pp/Portfile

```diff
@@ -3,9 +3,14 @@
 PortSystem          1.0
 PortGroup           cmake 1.1
 PortGroup           github 1.0
+PortGroup           legacysupport 1.1
+
+# https://trac.macports.org/ticket/69638
+legacysupport.newest_darwin_requires_legacy 16
+legacysupport.use_mp_libcxx yes
 
 github.setup        libSDL2pp libSDL2pp 0.18.1
-revision            0
+revision            1
```

Comment:
this is fixing a build failure and thus should not increase the `revision`

Reply:
Legacysupport linking may not leave binaries unchanged where they built.
(I am not gonna argue on this, but I think revbumping is right in this case. But I can also drop it.)

Reply:
okay, so that would possibly only affect < 10.6 - correct? In any case, it looks like this package builds in about 30 seconds on the buildbots so it's not a big deal. If you think it's safer to do that's fine with me.

Reply:
@reneeotten I rather keep revbumping as a matter of consistency. And yes, on 10.6 Intel it may matter (no prebuilt ports on ppc, obviously).